config-templates/google/gcp-compute-persistent-disk-csi-driver/1.8.0/storage-class-parameters.json

```diff
@@ -0,0 +1,46 @@
+[
+    {
+        "description": "The name of the storage class.",
+        "displayname": "Storage class name",
+        "name": "name",
+        "templateKeyName": [
+            "name"
+        ],
+        "required": "true",
+        "category": "config"
+    },
+    {
+        "description": "The disk type to use for the storage class.",
+        "displayname": "Disk type",
+        "name": "type",
+        "templateKeyName": [
+            "type"
+        ],
+        "required": "true",
+        "category": "config"
+    },
+    {
+        "description": "The volume binding mode.",
+        "displayname": "Volume binding mode",
+        "name": "volumeBindingMode",
+        "templateKeyName": [
+            "volumeBindingMode"
+        ],
+        "default": "WaitForFirstConsumer",
+        "required": "false",
+        "category": "config"
+    },
+    {
+        "description": "Specify a unique volume tier preference; in the range of 1-99 with 1 being the highest",
+        "displayname": "Storage Class Preference",
+        "name": "storage-class-preference",
+        "templateKeyName": [
+            "storage-class-preference"
+        ],
+        "required": "false",
+        "default": "99",
+        "category": "config",
+        "regex": "^[1-9]([0-9]?)$",
+        "type": "text"
+    }
+]
```

Comment:
Please add this
```
{
        "description": "Specify 'true' or 'false' to make the created storage class the default class.",
        "displayname": "Default Storage Class",
        "name": "is-default-class",
        "templateKeyName": [
            "storageclass.kubernetes.io/is-default-class"
        ],
        "default": "false",
        "required": "false",
        "category": "config",
        "type": "boolean"
    }
```

Reply:
Added here: https://github.com/IBM/ibm-satellite-storage/pull/221/commits/4a19a141fdc12733b26b78e0a2fe0713410875fd